test/e2e/manifest/testIPAddressAllocation/ipaddressallocation_privatetgw.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: crd.nsx.vmware.com/v1alpha1
+kind: IPAddressAllocation
+metadata:
+  name: guestcluster-workers-c
+spec:
+  ipAddressBlockVisibility: PrivateTGW
+  allocationSize: 32
```

Comment:
Could you set size 256 to mock TKG case?

Reply:
ok